### PR TITLE
chore: update list of contributors

### PR DESCRIPTION
**Pull request by Chriscbr**
*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Review by eladb — APPROVED** [review]

**Comment by eladb** [comment]
Periodic job?

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "build": "nx build winglang",
     "contributors:add": "all-contributors add",
     "contributors:check": "all-contributors check",
-    "contributors:generate": "all-contributors generate",
+    "contributors:update": "all-contributors check | grep \"Missing contributors\" -A 1 | tail -n1 | sed -e \"s/,//g\" | xargs -n1 | grep -v \"\\[bot\\]\" | xargs -n1 -I{} all-contributors add {} code",
```
Comment (MarkMcCulloh) [inline package.json:17]: image

Comment (MarkMcCulloh) [reply package.json:17]: What was wrong with `generate`?

**Review by MarkMcCulloh — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Chriscbr) [reply package.json:17]: 😆 

I'm not sure but `generate` didn't seem to actually create / update the list for me

**Comment by Chriscbr** [comment]
> Periodic job?

Good idea 👍

**Pull request merged**